modules/commit0.example.yml

```diff
@@ -0,0 +1,25 @@
+name: hello-world
+
+# Context will populated automatically or could be added manually
+context: 
+  awsCognito: 
+    poolID: 123
+    clientID: ABC
+
+modules: 
+  # module can be in any format the go-getter supports (path, github, url, etc.)
+  - source: './remote_templates/infrastructure'
+    params: 
+      apply: true
```

Comment:
Something like whether or not to apply should probably be controlled by the user at runtime.

Reply:
Yup that makes sense.